src/components/Field.vue

```diff
@@ -3,7 +3,13 @@
     <label v-if="label">{{ label }}</label>
     <p v-if="description">{{ description }}</p>
     <input v-if="type === 'text'" type="text" class="form-control" v-model="value" :placeholder="placeholder" :required="required">
+    <input v-if="type === 'email'" type="email" class="form-control" v-model="value" :placeholder="placeholder" :required="required">
     <textarea v-if="type === 'textarea'" class="form-control" v-model="value" :placeholder="placeholder" :required="required" :rows="rows || 4"></textarea>
+    <div class="options" v-if="options">
```

Comment:
Should we check that type is `checkbox` or `radio`? It seems unlikely now but if we had `{ type: 'wysiwyg', options }` then this could be a problem. Seems like a simple check to put in place and prevent a future problem.

Reply:
fixed